app/api/__init__.py

```diff
@@ -88,7 +88,7 @@
 api.route(SponsorRelationship, 'sponsor_event', '/sponsors/<int:id>/relationships/event')
 
 # tracks
-api.route(TrackList, 'track_list', '/tracks', '/events/<int:event_id>/tracks')
+api.route(TrackList, 'track_list', '/events/<int:event_id>/tracks')
 api.route(TrackDetail, 'track_detail', '/tracks/<int:id>', '/sessions/<int:session_id>/track')
```

Comment:
Event can have multiple tracks i.e 1-many so tracklist is the related view
Session can only be in a single track, that's why trackdetail for sessions